api/Window.json

```diff
@@ -153,15 +153,1930 @@
+              "version_added": null
+            },
+            "safari": {
+              "version_added": null
+            },
+            "safari_ios": {
+              "version_added": null
+            },
+            "samsunginternet_android": {
+              "version_added": null
+            },
+            "webview_android": {
+              "version_added": true
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": false,
+            "deprecated": true
+          }
+        }
+      },
+      "requestAnimationFrame": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Window/requestAnimationFrame",
+          "support": {
+            "chrome": [
+              {
+                "version_added": "24",
+                "notes": "The correct call in Chrome to cancel the request is currently <code>window.cancelAnimationFrame()</code>. For older versions, use <code>window.webkitCancelAnimationFrame()</code> & <code>window.webkitCancelRequestAnimationFrame()</code>; they have been deprecated but are still supported for now."
+              },
+              {
+                "version_added": "10",
+                "prefix": "webkit"
+              }
+            ],
+            "chrome_android": [
+              {
+                "version_added": "25",
+                "notes": "The correct call in Chrome to cancel the request is currently <code>window.cancelAnimationFrame()</code>. For older versions, use <code>window.webkitCancelAnimationFrame()</code> & <code>window.webkitCancelRequestAnimationFrame()</code>; they have been deprecated but are still supported for now."
+              },
+              {
+                "version_added": "18",
+                "prefix": "webkit"
+              }
+            ],
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": [
+              {
+                "version_added": "23",
+                "notes": "Callback parameter is a <code>DOMHighResTimestamp</code>. This means ten microsecond precision and zero time as <code>performace.now()</code>."
+              },
+              {
+                "version_added": "11",
+                "version_removed": "42",
+                "prefix": "moz",
+                "notes": "Callback parameter is a <code>DOMTimestamp</code>. This means millisecond precision and zero time as <code>Date.now()</code>."
+              },
+              {
+                "version_added": "4",
+                "version_removed": "11",
+                "prefix": "moz",
+                "notes": "Could be called with no input parameters."
+              }
+            ],
+            "firefox_android": [
+              {
+                "version_added": "23"
+              },
+              {
+                "version_added": "14",
```

Comment:
The page has "11" for this, is there a reason it's different here?

Reply:
Yes. Versions 1-3 were released in version 4, and 11-13 in version 14, [see here](https://github.com/mdn/browser-compat-data/pull/1715#issuecomment-378555080). The browser version schema reflects that (fails lint check).